example/SigningExample.js

```diff
@@ -7,19 +7,28 @@ var crypto = require('crypto');
 var app = express();
 app.use(express.static(require('path').join( __dirname + '/../')));
 
+// Add simple logging middleware
+app.use(function(req, res, next) {
+  console.log(req.method + ' ' + req.originalUrl);
+  next();
+});
+
 app.use(bodyParser.urlencoded({
 	extended: true
 }));
 
-app.listen(8080, '127.0.0.1');
+app.listen(8080, '127.0.0.1', function() {
```

Comment:
let's add a space after `function` in this anonymous. I know, I know, it's nit picky, but it's the standard I'm trying to maintain. Thanks!